runtime/compiler/control/CompilationThread.cpp

```diff
@@ -13035,12 +13036,18 @@ queryJITServerAOTCacheFilter(const char *methodSig, TR::Method::Type ty, TR::Com
 bool
 TR::CompilationInfo::methodCanBeJITServerAOTCacheStored(const char *methodSig, TR::Method::Type ty)
    {
-   return queryJITServerAOTCacheFilter(methodSig, ty, TR::Options::_JITServerAOTCacheStoreFilters);
+   return queryJITServerFilter(methodSig, ty, TR::Options::_JITServerAOTCacheStoreFilters);
    }
 
 bool
 TR::CompilationInfo::methodCanBeJITServerAOTCacheLoaded(const char *methodSig, TR::Method::Type ty)
    {
-   return queryJITServerAOTCacheFilter(methodSig, ty, TR::Options::_JITServerAOTCacheLoadFilters);
+   return queryJITServerFilter(methodSig, ty, TR::Options::_JITServerAOTCacheLoadFilters);
    }
+
+bool
+TR::CompilationInfo::isJITServerRemoteExclude(const char *methodSig, TR::Method::Type ty)
```

Comment:
If `queryJITServerFilter` responds `true`, then method can be compiled remotely. You have the condition reversed.
It's better to call this method `methodCanBeRemotelyCompiled` to look like the others.